lib/formatter/index.js

```diff
@@ -1,4 +1,4 @@
-const isNumber = require('is-number');
+const isNumber = (num) => (typeof num === "number" && num - num === 0) || (typeof num === "string" && Number.isFinite(+num) && num.trim() !== "");
```

Comment:
```suggestion
const isNumber = (num) => typeof num === "number" && num - num === 0;
```

Since we only use "isNumber" for number types — there's no need for the string check. Internally, this can be more efficient
```js
if (typeof value !== 'number' || !isNumber(value)) {
```